### DISCUSSION TITLE
httpx is using the python library instead of PD's httpx

### DISCUSSION THREAD

**Discussion by devaacode** [projectdiscovery/httpx#1587] [Q&A] [answered]
Ey, after i install httpx using go (the latest version), doesn't work.

" ❯ httpx -h
Error: Option '-h' requires 2 arguments." 
only say that.
sorry for my bad English

**Comment by olearycrew** [comment, marked as answer]
This is probably an existing installation of the python `httpx` library conflicting with PD's httpx

See https://docs.projectdiscovery.io/tools/httpx/overview#additional-httpx-resources and take a look at @MetzinAround explaining how to fix that here: https://www.youtube.com/watch?v=jpKFJmTs7iM

**Reply by GDATTACKER-RESEARCHER** [reply]
Sorry, but he is just removing the python httpx version but what about the tools like bbot which is a python module or baddns which require python httpx module and are well know in community for subdomains discovery

**Comment by bhaveshraj336** [comment]
@olearycrew  , These will mostly likely solve both the issues : 
adding these in ~/.bashrc:  
export PATH=$GOPATH/bin:$GOROOT/bin:$PATH

No Need to delete the python httpx library.
